src/pyxplor/pyxplor.py

```diff
@@ -0,0 +1,52 @@
+def time_series_explorer(input_df: pd.DataFrame, 
+                         date_column: str, 
+                         value_column: str, 
+                         freq: str = 'D', 

```

Comment:
Based on decision of @phchen5 's comment below re: removing frequency:
```suggestion
                         

```

Reply:
I didn't understand why frequency isn't important on time series analysis. Could you explain please?

Reply:
We don't have to worry too much about time frequency because we're just visualizing the data. Say we have a column of year values. [1999, 2001, 2010, 1998, 2012]. We will then just plot out the trend of another variable (or just the count of observations) over this period of time. The time will be on the x-axis. In this case, it simply wouldn't be possible to change the frequency of the date to something smaller right? Again, you can refer to the example of the outcome plot above.

Reply:
When I'm performing time series analysis, usually the data is static but I want to play around with the data range and frequency that I want to see in the plot. Sometimes I might want it daily or weekly, and some other times monthly or yearly

Reply:
yeah, I get it. I just thought maybe that could be a functionality that we can add later. But for now, let's try to keep it simple. Unless, you really want to do it, then go for it XDDD